src/frontend/extract-helpers/get-line-items.ts

```diff
@@ -4,11 +4,18 @@ import { FullExtractLineItem } from "./types";
 export const getLineItems = async (
   contractId: string,
   year: string,
-  month: string
+  month: string | number,
+  invoiceIsQuarterly: boolean = false
 ): Promise<FullExtractLineItem[]> => {
   const dashboardData = await getDashboardExtract();
+  const monthNumber = typeof month === "number" ? month : parseInt(month, 10);
```

Comment:
When is month a number?

Reply:
I mean always obviously but I thought they were strings in the view so they could be like "08" for August for example